src/main/java/com/openshift/internal/restclient/model/build/BuildRequest.java

```diff
@@ -36,7 +35,10 @@
 	private static final String REVISION_TYPE = REVISION + "." + TYPE;
 	private static final String TRIGGERED_BY = "triggeredBy";
 	private static final String MESSAGE = "message";
-	
+	private static final String ENV = "env";
+	private static final String ENV_NAME = "name";
```

Comment:
NAME and VALUE are available as part of the interface implemented by KubernetesResource